testing/consul/APKBUILD

```diff
@@ -60,14 +36,18 @@ prepare() {
 }
 
 build() {
-	cd "$_builddir"/$pkgname || return 1
-	export GOPATH="$srcdir"
-	go build -v -o bin/consul \
-		-ldflags "-X main.GitDescribe $pkgver" || return 1
+	cd "$builddir"/$pkgname || return 1
+
+	mkdir -p "${builddir}/src/github.com/hashicorp"
+	ln -s "${builddir}" "${builddir}/src/github.com/hashicorp/consul"
+	cd "${builddir}/src/github.com/hashicorp/consul"
```

Comment:
@gena01 this wasn't the point. I know that go needs a specific directory structure to work with, and an exported `$GOPATH` to the root of that directory structure; my question is about a choice we make, to minimize the amount of divergent ways of doing it, because I am doing it differently, and maybe others do too.